frontend/src/__tests__/cypress/cypress/e2e/applications/enabled.cy.ts

```diff
@@ -0,0 +1,34 @@
+import { mockDashboardConfig } from '~/__mocks__/mockDashboardConfig';
+import { mockDscStatus } from '~/__mocks__/mockDscStatus';
+import { mockStatus } from '~/__mocks__/mockStatus';
+import { mockComponents } from '~/__mocks__/mockComponents';
+import { enabledPage } from '~/__tests__/cypress/cypress/pages/enabled';
+import { jupyterCard } from '~/__tests__/cypress/cypress/pages/components/JupyterCard';
+
+describe('Enabled Page', { testIsolation: false }, () => {
+  beforeEach(() => {
+    cy.intercept('/api/dsc/status', mockDscStatus({}));
+    cy.intercept('/api/status', mockStatus());
+    cy.intercept('/api/config', mockDashboardConfig({}));
+    cy.intercept('/api/components?installed=true', mockComponents());
+  });
+
+  it('check enabled page', () => {
+    enabledPage.visit();
+    enabledPage.findPageTitle().should('have.text', 'Enabled');
+    enabledPage.shouldHavePageDescription();
```

Comment:
Asserting the presence of _static text_ doesn't seem necessary to me. It sets a bad precedent that we should assert all text on every page. We should focus on actions and dynamic content.

I would remove this test and add the visit call to the next test.